docs/api-guide/routers.md

```diff
@@ -65,7 +65,7 @@ For example, you can append `router.urls` to a list of existing views…
 
     urlpatterns += router.urls
 
-Alternatively you can use Django's `include` function, like so…
+Alternatively you can use Django's `include` function, like so...
```

Comment:
Why don't you like `…`?

Reply:
Oh - that was unintentional. I have a plugin that converts magic quotes to their ascii equivalent, but it also catches ellipses apparently.